SDK v3 Examples/ios-sdk-app/ios-sdk-app/ViewController+MPIMapViewDelegate.swift

```diff
@@ -87,16 +87,54 @@ extension ViewController: MPIMapViewDelegate {
 //        updateBlueDotBanner(blueDot: blueDot)
     }
 
-    func onDataLoaded(data: MPIData) {}
+    func onDataLoaded(data: MPIData) {
+        let rankings: MPIRankings? = data.rankings
+        let polygonRank = data.polygons[0].rank
```

Comment:
These don't seem to do anything

Reply:
I'm just trying to show that rankings information can be accessed now. Should I leave it out?

Reply:
A variable that is declared and never read is the sort of thing a linter would catch, it might make someone using the SDK think they need to read that data for some reason. Like the rankings aren't calculated unless you access that getter or something. I would maybe make those lines comments, with another explanation as to why you might want to access those rankings. Or at the very least console log the data.

Reply:
I see. I'm going to remove those lines entirely since they're already in the release notes.